mithril-aggregator/src/http_server/routes/signer_routes.rs

```diff
@@ -175,29 +195,29 @@ mod handlers {
     pub async fn registered_signers(
         registered_at: String,
         logger: Logger,
+        epoch_service: EpochServiceWrapper,
         verification_key_store: Arc<dyn VerificationKeyStorer>,
     ) -> Result<impl warp::Reply, Infallible> {
-        debug!(logger, "GET /signers/registered/{registered_at:?}");
```

Comment:
I removed it because it's already covered by the global log system on the http server. Ie, those lines appears in the logs:
```json
{  "msg": "GET /aggregator/signers/registered/16 200 OK" },
{  "msg": "GET /aggregator/signers/registered/latest 200 OK" }  
```